server2/Arcadia.Assistant/Arcadia.Assistant.ExternalStorages.Abstractions/StorageItem.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace Arcadia.Assistant.ExternalStorages.Abstractions
+{
+    using System;
+    using System.Text.Json.Serialization;
+
+    public class StorageItem
+    {
+        [JsonPropertyName("Id")]
```

Comment:
Could you use `DataContract` and `DataMember` instead? They are more common as `Json` ones.

Reply:
Its impossible. Text.Json serialization support JsonPropertyName attrubte only for change default scheme.